draft-ietf-rats-architecture.md

```diff
@@ -1492,20 +1491,29 @@ in terms of `time(RX_v)-time(RG_v)`, then the Relying Party can check
 
 ## Example 3: Handle-based Passport Model Example
 
-The following example illustrates a hypothetical Passport Model
-solution that uses handles instead of nonces or timestamps.
-In this example, Evidence generation based on received handles always uses the current (most recent) handle.
-As handles are distributed over the network, all involved entities receive a fresh handle at roughly the same time.
-Due to distribution over the network, there is some jitter with respect to the time the Handle is received, time(HR), for each involved entity.
-To compensate for this jitter, there is a small period of overlap (a specified offset) in which both a current handle and corresponding former handle are valid in Evidence appraisal: `validity-duration = time(HR'_v) + offset - time(HR_v)`. The offset is typically based on a network's round trip time.
-Analogously, the generation of valid Evidence is only possible, if the age of the handle used is lower than the validity-duration: `time(HR_v) - time(EG_a) < validity-duration`.
+The example in {{fig-handle-passport}} illustrates a Handle-based interaction
+using the passport topology model.
+
+The Handle Distributor broadcasts handle `H` which starts a new
+epoch `E` for a protocol participant upon reception at `time(HR)`.
 
-From the point of view of a Verifier, the generation of valid Evidence is only possible, if the age of the handle used in the Evidence generation is younger than the duration of the distribution interval -- "(time(HR'_v)-time(HR_v)) - (time(HR_a)-time(EG_a)) \< validity-duration".
+The Attester generates Evidence incorporating handle `H` and conveys it to the
+Verifier.
 
-Due to the validity-duration of handles, multiple different pieces of Evidence can be generated based on the same handle.
-The resulting granularity (time resolution) of Evidence freshness is typically lower than the resolution of clock-based tickcounters.
+The Verifier appraises that the received handle `H` is "fresh" according to the
+definition provided in {{freshness}}, and generates an Attestation Result.  The
```

Comment:
I'd recommend referencing the "Implicit Timekeeping using Epoch Handles" subsection rather than the full "Freshness" section.  And "according to the definition provided" there is ambiguous since it mentions that any solution has to take one of three approaches, so need to clarify which approach is shown in this example.

I.e., quoting text there:
> An actual solution also needs to take into account race conditions
> when transitioning to a new epoch, such as by using a counter signed
> by the Handle Distributor as the handle, or by including both the current and
> previous handles in messages and/or checks, by requiring retries
> in case of mismatching handles, or by buffering incoming messages
> that might be associated with a handle that the receiver has not yet
> obtained.